Source/WebKit/UIProcess/Automation/SimulatedInputDispatcher.h

```diff
@@ -59,10 +59,11 @@ using VirtualKeyMap = HashMap<VirtualKey, VirtualKey, WTF::IntHash<VirtualKey>,
 #if ENABLE(WEBDRIVER_KEYBOARD_GRAPHEME_CLUSTERS)
 // A CharKey must only ever represent a single unicode codepoint or a single grapheme cluster.
 using CharKey = String;
+using CharKeySet = ListHashSet<String>;
```

Comment:
This change is suspicious.  Why are we changing from a 32-bit type to an allocated String?

Reply:
We are not. The old code did `ListHashSet<CharKey>`, and `CharKey` here is `String`.

Reply:
oh, right